frontend/src/utils/poll.utils.ts

```diff
@@ -372,8 +372,10 @@ export const checkPollPermission = async (
     } catch (e) {
       const problem = e as any
       error = problem.error?.message ?? problem.reason ?? problem.code ?? problem
-      console.log('Error when testing permission to vote on', proposalId, ':', error)
+      console.error('Error when testing permission to vote on', proposalId, ':', error)
+      console.error('proof:', proof)
       canVote = denyWithReason(`there was a technical problem verifying your permissions`)
+      return
```

Comment:
Doing this would cause the permission check never to complete, i.e. the spinner keeps spinning, no message is displayed on the UI, no way for the user to trigger re-testing.

Therefore I changed the implementation how we achieve this (not storing failed checkes).